composer.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "friendsoftypo3/form-legacy",
 	"type": "typo3-cms-extension",
-	"description": "Legacy version 7 of ext:form",
+	"description": "Fork from friendsoftypo3/form-legacy / Test to enable it for 9.x ",
```

Comment:
maybe we shouldn't merge that ;)

Reply:
maybe with a new version just for v9 in an extra branch?